atomic_reactor/tasks/binary_container_build.py

```diff
@@ -98,7 +178,23 @@ def build_container(
         This method returns an iterator which yields individual lines from the stdout
         and stderr of the build process as they become available.
         """
-        build_cmd = ["/bin/sh", "-c", "for i in 1 2 3; do echo output $i; sleep 0.1; done"]
+        cli_buildargs = [f"--build-arg={key}={value}" for key, value in build_args.items()]
+        build_cmd = [
+            *self._podman_remote_cmd,
+            "build",
+            f"--tag={dest_tag}",
+            str(build_dir.path),
+            "--memory-swap=0",  # TBD: why is this important?
```

Comment:
@lkolacek @MartinBasti I found this in design, but I'm not actually share what it does or why we want it

Reply:
since we cannot use --memory limits now it's useless and can be dropped.

Reply:
Ack, will remove